README.adoc

```diff
@@ -65,10 +59,10 @@ image::img/hawkular-metric-charts.png[Hawkular Metrics Chart Type]
 
 _Sample code for this chart:_
 
-[source,javascript]
+[source,html]
 ----
 <hk-metric-chart
-    data="dataPoints"
+    raw-data="dataPoints"
```

Comment:
Yes, I found that was clearer to split data model to "stats" and "raw" data points, rather than having a much more polymorph class.